tests/test_base.py

```diff
@@ -447,6 +457,23 @@ def test_gettimedelta_defaults_raises(self):
         self.assertRaises(AssertionError, getter.gettimedelta, 'test', 4.2)
         self.assertRaises(AssertionError, getter.gettimedelta, 'test', (1, ))
 
+    def test_getpath_defaults(self):
+        """Test fetching a pathlib.Path"""
+        getter = getconf.ConfigGetter('TESTNS')
+        self.assertEqual(Path(''), getter.getpath('test'))
```

Comment:
```suggestion
        self.assertEqual(Path('.'), getter.getpath('test'))
```

Let's be explicit here too. :)